opal_playground_rspec.org

```diff
@@ -34,25 +34,32 @@ The remainder of this article shows you how.
 
 * Solution
 #+ATTR_HTML: :target "_blank"
-Like the =jsbin=-like workspace, [[http://fkchang.github.io/opal-playground][opal-playground]] has a [[http://fkchang.github.io/opal-playground/rspec/][page]]
-configured to run spec tests by pasting them in and executing. Now instead of sending
-my co-workers snippets of code, I test the spec in the Opal playground and test it.
-When I'm satisfied that it's doing what I want, I send the link directly
-to him.  Here's what it looks like:
+Aside from the =jsbin=-like workspace, [[http://fkchang.github.io/opal-playground][opal-playground]] has a [[http://fkchang.github.io/opal-playground/rspec/][rspec
+page]] configured to run spec tests by writing or pasting them in and
+executing. Now instead of sending my co-workers snippets of code, I
+test the spec in the Opal playground and test it.  When I'm satisfied
+that it's doing what I want, I send the link directly to him.  Here's
+what I originally pasted to him:
 #+ATTR_HTML: :target "_blank"
-[[http://git.io/vmalc][(direct link]] if below iframe doesn't look right).  You can vertically scroll
+(I've embedded the actual rspec page in an iframe [[http://git.io/vmalc][direct link]] if below
+iframe doesn't look right in your browser).  You can vertically scroll
 the text pane to see the remaining *rspec* content.
 
 #+BEGIN_HTML
 <iframe width="100%" height="500" src="http://git.io/vmalc" frameborder="0" allowfullscreen></iframe>
```

Comment:
I'm having trouble scrolling to line 1 in the Opal code; it won't scroll above line 4 for me in both **Chrome** and **Firefox**.  I tried setting the above `height` to **550** and **600** to no avail.  Could you check; maybe you have better ideas than I.